fedora/jellyfin-web.spec

```diff
@@ -0,0 +1,43 @@
+%global         debug_package %{nil}
+
+Name:           jellyfin-web
+Version:        10.6.0
+Release:        1%{?dist}
+Summary:        The Free Software Media System web client
+License:        GPLv3
+URL:            https://jellyfin.media
+# Jellyfin Server tarball created by `make -f .copr/Makefile srpm`, real URL ends with `v%{version}.tar.gz`
```

Comment:
```suggestion
Release:        1%{?dist}
Summary:        Web Client for Jellyfin
License:        GPLv3
URL:            https://jellyfin.org
```

Reply:
I',m trying to keep that lead line the same in all packages.